easybuild/easyblocks/s/suitesparse.py

```diff
@@ -172,8 +184,25 @@ def configure_step(self):
 
         else:
             # after v6.0.0, no option for metis, its own metis is used anyway
-            # nothing to do here, set the CMAKE_OPTIONS in easyconfigs
-            pass
+            # set CMAKE_OPTIONS if it is not specified in easyconfigs
+            # CMAKE_INSTALL_PREFIX is managed by easybuild
+            cmake_options = '-DCMAKE_INSTALL_PREFIX=%s' % self.installdir
+
+            flexiblas = get_software_root('FlexiBLAS')
+            mkl = get_software_root('imkl')
+            openblas = get_software_root('OpenBLAS')
+
+            if flexiblas:
+                cmake_options = ' '.join([cmake_options, '-DBLA_VENDOR=FlexiBLAS'])
```

Comment:
We should first check whether `-DBLA_VENDOR=` is specified in `cmake_options` already?

If so, we should trust the easyconfig, and not derive the value to use based on BLAS/LAPACK library being used.